src/__global.zig

```diff
@@ -92,10 +92,14 @@ pub fn runExitCallbacks() void {
 
 /// Flushes stdout and stderr and exits with the given code.
 pub fn exit(code: u8) noreturn {
+    exitWide(@as(u32, code));
+}
+
+pub fn exitWide(code: u32) noreturn {
     runExitCallbacks();
     Output.flush();
     std.mem.doNotOptimizeAway(&Bun__atexit);
-    std.c.exit(code);
+    std.c.exit(@bitCast(code));
```

Comment:
This will fail to compile on Linux, no?

Reply:
I think you want `@intCast`

Reply:
std.c.exit takes a `c_int` on all platforms I think

Reply:
or wait, maybe not